label_studio_converter/converter.py

```diff
@@ -360,8 +360,12 @@ def convert_to_coco(self, input_data, output_dir, output_image_dir=None, is_dir=
         else:
             output_image_dir = os.path.join(output_dir, 'images')
             os.makedirs(output_image_dir, exist_ok=True)
-        images, categories, annotations = [], [], []
-        category_name_to_id = {}
+        images, annotations = [], []
+        categories = [
+            {"id": n, "name": x} for n, x in enumerate(self._schema["label"]["labels"])
```

Comment:
Hi, @daavoo ! Thanks for your PR. Not sure about this line - looks like it's hardcoded tag name `"label"` - I guess it's better to replace it with `self._output_tags[0]`